witchcraft/server_metrics.go

```diff
@@ -141,14 +141,31 @@ func (s *Server) initMetrics(ctx context.Context, installCfg config.Install) (rR
 	}, nil
 }
 
-func (s *Server) filterMetricValues(metricID string, metricVal metrics.MetricVal) map[string]interface{} {
+func (s *Server) collectMetricValues(metricName string, metricVal metrics.MetricVal) map[string]interface{} {
+	valuesToUse := make(map[string]interface{})
```

Comment:
nit, but for new code, slight preference for `any` over `interface{}` (but don't feel strongly about this, especially given that `interface{}` is still used in other existing code in this file)

```suggestion
func (s *Server) collectMetricValues(metricName string, metricVal metrics.MetricVal) map[string]any {
	valuesToUse := make(map[string]any)
```